sources/Renderer/Direct3D12/D3D12RenderSystem.cpp

```diff
@@ -19,19 +19,19 @@
 #include <LLGL/Utils/ForRange.h>
 #include <LLGL/Backend/Direct3D12/NativeHandle.h>
 #include <limits.h>
-#include <codecvt>
 
 #include "Buffer/D3D12Buffer.h"
 #include "Buffer/D3D12BufferArray.h"
 #include "Buffer/D3D12BufferConstantsPool.h"
 
 #include "Texture/D3D12MipGenerator.h"
 
-#include "RenderState/D3D12GraphicsPSO.h"
 #include "RenderState/D3D12ComputePSO.h"
+#include "RenderState/D3D12GraphicsPSO.h"
 
 #include <LLGL/Backend/Direct3D12/NativeHandle.h>
 
+#include <dxgi1_5.h>
```

Comment:
What minimum Windows SDK version requires DXGI 1.5? So far DXGI 1.4 was the highest used DXGI version in the D3D12 backend, which I set the WinSDK version to 10.0.10240.0 in the [docs page](https://github.com/LukasBanana/LLGL/tree/master/docu#windows). Would this require to lift the SDK version or can we wrap this into an `#if LLGL_D3D12_ENABLE_FEATURELEVEL` block like with the D3D11 backend? Having that said, I think I'll have to split this macro up anyway, since feature level, D3D version, DXGI version, and shader-model are all separate concepts that are not tied together 1:1, but right now all of these things are wrapped under the same CMake option and macro.

Reply:
According to [this page](https://github.com/toivjon/dxgi-sandbox), it's Windows 10.